modules/lnd.nix

```diff
@@ -155,6 +181,8 @@ in {
         ${optionalString cfg.announce-tor "echo externalip=$(cat /var/lib/onion-chef/lnd/lnd) >> '${cfg.dataDir}/lnd.conf'"}
       '';
       serviceConfig = nix-bitcoin-services.defaultHardening // {
+        RuntimeDirectory = "lnd";
+        RuntimeDirectoryMode = "710";
```

Comment:
At first glance, permission `g=x` won't allow access to btcpayserver. Have you tested that this works?

Reply:
`g=x` allows directory read access to users in the lnd group, read permissions on custom macaroons are then individually handled.

Reply:
Yes, you're right! :thumbsup: